app/assets/javascripts/arclight/oembed_controller.js

```diff
@@ -12,32 +12,23 @@ export default class extends Controller {
       return
     }
 
-    fetch(this.urlTarget)
-      .then((response) => response.text())
+    fetch(this.urlValue)
+      .then((response) => {
+        if (response.ok) return response.text()
+        throw new Error(`HTTP error, status = ${response.status}`)
+      })
       .then((body) => {
-        const oEmbedEndPoint = this.findOEmbedEndPoint(body)
+        const oEmbedEndPoint = OembedController.findOEmbedEndPoint(body)
         if (!oEmbedEndPoint || oEmbedEndPoint.length === 0) {
-          console.warn(`No oEmbed endpoint found in <head> at ${this.urlTarget}`)
+          console.warn(`No oEmbed endpoint found in <head> at ${this.urlValue}`)
           return
         }
         this.loadEndPoint(oEmbedEndPoint)
+      }).catch(error => {
+        console.error(error)
       })
   }
 
-  // Convert data-arclight-oembed-* attributes to URL parameters for the viewer
-  static normalizeParams(attributes) {
-    return Object.keys(attributes).reduce((acc, attribute) => {
-      // Reverse data attribute name conversion. See:
-      // https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#name_conversion
-      const parameterName = attribute.replace('arclightOembed', '')
-        .replace(/([a-z])([A-Z])/g, '$1-$2')
-        .toLowerCase()
-
-      acc[parameterName] = attributes[attribute]
-      return acc
-    }, {})
-  }
-
```

Comment:
why are we getting rid of this? we need it to parse the extra params, which is what powers in-document search in VT: https://github.com/sul-dlss/vt-arclight/pull/451

did we change the implementation of this to drop support for passing extra params to the viewer?

Reply:
It didn't look like this did anything. Maybe it needs a test?

Reply:
Why did support go away? Does stimulus prevent us from using arbitrary `data-*` attributes?

If we need this PR in order to fix anything in VT, we should be aware that upgrading our arclight version will break in-document search until and unless we reimplement it there.

Reply:
It went away because of the need to fix https://github.com/projectblacklight/arclight/issues/1401 and because it wasn't clear to me that it was used for anything (not tested and the documentation was unclear about the need for it.)